### PR TITLE
Update DLRS guide for v6 release

### LINKED ISSUE #1122: Update DLRS guide page for v6 release
**Describe the error/improvement to an existing document or image**
New features and updated component versions need to be added to guides/stacks/dlrs.rst

**Additional context**
Release planned 4-21-20

### PR DESCRIPTION
**Pull request by bd-dean**
Please publish Tuesday April 21.  Closes #1122

### CODE REVIEW

File: source/guides/stacks/dlrs.rst [hunk 1]
```diff
@@ -20,16 +20,17 @@ customized solutions, and enables you to quickly prototype and deploy Deep
...
 solution.
 
-The latest release of the Deep Learning Reference Stack (`DLRS V5.0`_ ) supports the following features:
+The latest release of the Deep Learning Reference Stack (`DLRS V6.0`_ ) supports the following features:
 
-* TensorFlow* 1.15 and TensorFlow* 2.0, an end-to-end open source platform for machine learning (ML).
+* TensorFlow* 1.15 and TensorFlow* 2.2.0, an end-to-end open source platform for machine learning (ML).
 * PyTorch* 1.3, an open source machine learning framework that accelerates the path from research prototyping to production deployment.
 * PyTorch Lightning* which is a lightweight wrapper for PyTorch designed to help researchers set up all the boilerplate state-of-the-art training.
-* Transformers* , a state-of-the-art Natural Language Processing (NLP) for TensorFlow 2.0 and PyTorch.
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:28]: * (NLP) library

File: source/guides/stacks/dlrs.rst [hunk 2]
```diff
@@ -20,16 +20,17 @@ customized solutions, and enables you to quickly prototype and deploy Deep
...
-* Intel® OpenVINO™ model server version 2019_R3, delivering improved neural network performance on Intel processors, helping unlock cost-effective, real-time vision applications.
+* Transformers* , a state-of-the-art Natural Language Processing (NLP) for TensorFlow 2.0 and Flair for PyTorch.
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:28]: May be change it to:
* Transformers* , a state-of-the-art Natural Language Processing (NLP) library for TensorFlow 2.0 and PyTorch 
* Flair*, a PyTorch NLP framework

File: source/guides/stacks/dlrs.rst [hunk 3]
```diff
@@ -20,16 +20,17 @@ customized solutions, and enables you to quickly prototype and deploy Deep
...
+* OpenVINO™ model server version 2020.1, delivering improved neural network performance on Intel processors, helping unlock cost-effective, real-time vision applications.
 * Intel Deep Learning Boost (DL Boost) with AVX-512 Vector Neural Network Instruction (Intel AVX-512 VNNI) designed to accelerate deep neural network-based algorithms.
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:30]: nit : Intel® AVX...

File: source/guides/stacks/dlrs.rst [hunk 4]
```diff
@@ -20,16 +20,17 @@ customized solutions, and enables you to quickly prototype and deploy Deep
 Learning workloads. Use this guide to run benchmarking workloads on your
 solution.
 
-The latest release of the Deep Learning Reference Stack (`DLRS V5.0`_ ) supports the following features:
+The latest release of the Deep Learning Reference Stack (`DLRS V6.0`_ ) supports the following features:
 
-* TensorFlow* 1.15 and TensorFlow* 2.0, an end-to-end open source platform for machine learning (ML).
+* TensorFlow* 1.15 and TensorFlow* 2.2.0, an end-to-end open source platform for machine learning (ML).
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:25]: to be specific: TensorFlow* 2.2.0(rc1)

File: source/guides/stacks/dlrs.rst [hunk 5]
```diff
@@ -20,16 +20,17 @@ customized solutions, and enables you to quickly prototype and deploy Deep
...
 * PyTorch* 1.3, an open source machine learning framework that accelerates the path from research prototyping to production deployment.
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:26]: nit: PyTorch 1.4 *

File: source/guides/stacks/dlrs.rst (hunk 2 above)
Comment (bd-dean) [reply source/guides/stacks/dlrs.rst]: good call -- they're not married. :)

File: source/guides/stacks/dlrs.rst [hunk 6]
```diff
@@ -651,20 +653,20 @@ In this example, you will:
...
 This example walks through the basic instructions for using the inference engine.
 
 #. Starting the Model Server
 
    The process is similar to how we start `Jupter notebooks` on our containers
 
-   Run this command to spin up a OpenVino model fetched from GCP
+   Run this command to spin up a OpenVINO™ toolkit model fetched from GCP
 
    .. code-block:: bash
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:671]: @bd-dean  the model storage path has been changed, let's update the model path,
from:  --model_path gs://intelai_public_models/resnet_50_i8  
to : --model_path gs://public-artifacts/intelai_public_models/resnet_50_i8

File: source/guides/stacks/dlrs.rst [hunk 7]
```diff
@@ -1090,6 +1095,8 @@ Related topics
 
 .. _DLRS V5.0: https://clearlinux.org/blogs-news/deep-learning-reference-stack-v50-now-available
 
+.. _DLRS V6.0: https://clearlinux.org/blogs-news/deep-learning-reference-stack-v6.0-now-available
+
 .. _dlrs-tfjob: https://github.com/clearlinux/dockerfiles/tree/master/stacks/dlrs/kubeflow/dlrs-tfjob
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:1100]: nit: update this url to github.com/intel/stacks

File: source/guides/stacks/dlrs.rst [hunk 8]
```diff
@@ -1079,15 +1079,15 @@ OpenVINO is a trademark of Intel Corporation or its subsidiaries
...
 
-.. _Eigen: https://hub.docker.com/r/clearlinux/stacks-dlrs-oss/
+.. _Eigen: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-clearlinux/
 
-.. _Intel MKL-DNN: https://hub.docker.com/r/clearlinux/stacks-dlrs-mkl/
+.. _Intel MKL-DNN: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-clearlinux/
 
 .. _PyTorch with OpenBLAS: https://hub.docker.com/r/clearlinux/stacks-pytorch-oss
 
 .. _PyTorch with Intel MKL-DNN: https://hub.docker.com/r/clearlinux/stacks-pytorch-mkl
```
Comment (unrahul) [inline source/guides/stacks/dlrs.rst:1088]: nits:
TensorFlow Eigen:  sysstacks/dlrs-tensorflow-clearlinux:v0.6.0-oss
Tensorflow MKL-DNN: 
- 1.15 - sysstacks/dlrs-tensorflow-clearlinux:v0.6.0
- 2.2.0 - sysstacks/dlrs-tensorflow2-clearlinux:v0.6.0

There is only one version of Eigen based image for TF

PyTorch with OpenBLAS - sysstacks/dlrs-pytorch-clearlinux:v0.6.0-oss
PyTorch with Intel MKL-DNN - sysstacks/dlrs-pytorch-clearlinux:v0.6.0

File: source/guides/stacks/dlrs.rst [hunk 9]
```diff
@@ -20,19 +20,21 @@ customized solutions, and enables you to quickly prototype and deploy Deep
...
+* Flair*, a PyTorch NLP framework
+* OpenVINO™ model server version 2020.1, delivering improved neural network performance on Intel processors, helping unlock cost-effective, real-time vision applications.
+* Intel Deep Learning Boost (DL Boost) with AVX-512 Vector Neural Network Instruction (Intel® AVX-512 VNNI), designed to accelerate deep neural network-based algorithms.
 * Deep Learning Compilers (TVM* 0.6), an end-to-end compiler stack.
 
+
 .. important::
 
-   To take advantage of the Intel® AVX-512 and VNNI functionality (including the MKL-DNN releases)  with the Deep
+   To take advantage of the Intel® AVX-512 and VNNI functionality (including the `oneDNN`_ releases)  with the Deep
```
Comment (mvincerx) [inline source/guides/stacks/dlrs.rst:37]: On first instance, pleaser use full name: 
```
Intel® oneAPI Deep Neural Network Library (oneDNN)
```
Thereafter, use only: `oneDNN`

You can append that sentence and thereby preserve the hyperlink by adding:
"found at ``oneDNN_ `` releases"

File: source/guides/stacks/dlrs.rst [hunk 10]
```diff
@@ -44,7 +46,7 @@ Releases
 
 Refer to the `Deep Learning Reference Stack website`_ for information and download links for the different versions and offerings of the stack.
```
Comment (mvincerx) [inline source/guides/stacks/dlrs.rst:47]: Ln 40: 
Correct usage is:  
Intel® Xeon® Scalable processor; or 
Intel® Xeon® Scalable processor family

It depends on what you intend. I believ you mean the first.

File: source/guides/stacks/dlrs.rst [hunk 11]
```diff
@@ -20,19 +20,21 @@ customized solutions, and enables you to quickly prototype and deploy Deep
...
 * PyTorch Lightning* which is a lightweight wrapper for PyTorch designed to help researchers set up all the boilerplate state-of-the-art training.
-* Transformers* , a state-of-the-art Natural Language Processing (NLP) for TensorFlow 2.0 and PyTorch.
-* Intel® OpenVINO™ model server version 2019_R3, delivering improved neural network performance on Intel processors, helping unlock cost-effective, real-time vision applications.
-* Intel Deep Learning Boost (DL Boost) with AVX-512 Vector Neural Network Instruction (Intel AVX-512 VNNI) designed to accelerate deep neural network-based algorithms.
+* Transformers* which is a state-of-the-art Natural Language Processing (NLP) library for TensorFlow 2.0 and PyTorch
+* Flair*, a PyTorch NLP framework
+* OpenVINO™ model server version 2020.1, delivering improved neural network performance on Intel processors, helping unlock cost-effective, real-time vision applications.
+* Intel Deep Learning Boost (DL Boost) with AVX-512 Vector Neural Network Instruction (Intel® AVX-512 VNNI), designed to accelerate deep neural network-based algorithms.
```
Comment (mvincerx) [inline source/guides/stacks/dlrs.rst:31]: Ln 31. Correct usage: 

Intel® Deep Learning Boost (Intel® DL Boost)

File: source/guides/stacks/dlrs.rst [hunk 12]
```diff
@@ -574,12 +576,12 @@ From the browser, you will see the following notebooks.
 This example along with the other notebooks show how to get up and running with Transformers.  More detail on using Transformers* is available through the `Transformers`_ github repository.
 
 
-Using the Intel® OpenVINO Model Optimizer
-*****************************************
-
-The Intel OpenVINO toolkit has two primary tools for deep learning, the inference engine and the model optimizer. The inference engine is integrated into the Deep Learning Reference Stack. It is better to use the model optimizer after training the model, and before inference begins. This example will explain how to use the model optimizer by going through a test case with a pre-trained TensorFlow model.
+Using the OpenVINO™ Model Optimizer
+***********************************
+.
```
Comment (mvincerx) [inline source/guides/stacks/dlrs.rst:581]: Remove `.`

File: source/guides/stacks/dlrs.rst [hunk 13]
```diff
@@ -1007,8 +1009,8 @@ To stop the container, execute the following from your host system:
    .. code-block:: console
 
       # docker rmi 827
-      Untagged: clearlinux/stacks-dlrs-oss:latest
-      Untagged: clearlinux/stacks-dlrs-oss@sha256:381f4b604537b2cb7fb5b583a8a847a50c4ed776f8e677e2354932eb82f18898
+      Untagged: sysstacks/dlrs-tensorflow-clearlinux:latest
+      Untagged: sysstacks/dlrs-tensorflow-clearlinux@sha256:381f4b604537b2cb7fb5b583a8a847a50c4ed776f8e677e2354932eb82f18898
       Deleted: sha256:82757ec1648a906c504e50e43df74ad5fc333deee043dbfe6559c86908fac15e
```
Comment (mvincerx) [inline source/guides/stacks/dlrs.rst:1014]: Ln 1027L
It's not clear in this header if you mean "OpenMPI", referenced above, or if you mean something like:
`Cluster OpenMP* product for Intel® C++ Compiler for Linux* OS`

Please correct this header.

Comment (bd-dean) [reply source/guides/stacks/dlrs.rst:1013]: Corrected header and rewrote initial paragraph for clarity. I hope.

File: source/guides/stacks/dlrs.rst (hunk 10 above)
Comment (bd-dean) [reply source/guides/stacks/dlrs.rst:46]: I did mean the first. TY!

File: source/guides/stacks/dlrs.rst (hunk 13 above)
Comment (mvincerx) [reply source/guides/stacks/dlrs.rst:1013]: Got it. Thank you.

**Comment by mvincerx** [comment]
LGTM. 
Ready for merge and pub?

**Comment by unrahul** [comment]
@mvincerx yes, please go ahead.

**Pull request merged**